src/python/cli_new/lib/cli/mesos.py

```diff
@@ -413,6 +417,7 @@ def _wait(self):
             'wait_container': {
                 'container_id': self.container_id}}
         req_extra_args = {
+            'verify': self.config.agent_ssl_verify(),
```

Comment:
What happens if `elf.config.agent_ssl_verify()` is `None`, will it work?

Reply:
Thats obsolete now :-)